backend/open_webui/migrations/versions/e0cd6c95cb25_add_time_saved_in_seconds_column_to_.py

```
@@ -0,0 +1,38 @@
+"""Add time_saved_in_seconds column to completion table
+
+Revision ID: e0cd6c95cb25
+Revises: 8b04db1441b0
+Create Date: 2025-02-25 08:51:45.692183
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+import open_webui.internal.db
```

Ruff (0.8.2)

`open_webui.internal.db` imported but unused

Remove unused import: `open_webui.internal.db`

(F401)